src/Nri/Ui/Panel/V1.elm

```diff
@@ -0,0 +1,181 @@
+module Nri.Ui.Panel.V1 exposing
+    ( view, Attribute
+    , header
+    , plaintext, markdown, html
+    , containerCss, headerCss, css
+    , primary, secondary
+    )
+
+{-| Create panels (AKA wells.)
+
+@docs view, Attribute
+
+
+## Content
+
+@docs header
+@docs plaintext, markdown, html
+@docs containerCss, headerCss, css
+
+
+## Theme
+
+@docs primary, secondary
+
+-}
+
+import Css exposing (Style)
+import Html.Styled exposing (..)
+import Html.Styled.Attributes as Attributes
+import Markdown
+import Nri.Ui.Colors.V1 as Colors
+import Nri.Ui.Fonts.V1 as Fonts
+
+
+type alias Config msg =
+    { header : String
+    , content : List (Html msg)
+    , theme : Theme
+    , css : List Style
+    , headerCss : List Style
+    , containerCss : List Style
+    }
+
+
+defaultConfig : Config msg
+defaultConfig =
+    { header = ""
+    , content = []
+    , theme = Primary
+    , css = []
+    , headerCss = []
+    , containerCss = []
+    }
+
+
+{-| -}
+type Attribute msg
+    = Attribute (Config msg -> Config msg)
```

Comment:
I've tried to explain this function-passing style to folks during onboarding and it's always confusing. Could this maybe just be a custom type with a reducer, for that reason?

Reply:
I think theoretically folks shouldn't need to learn this pattern when they're onboarding, since it's mostly used in noredink-ui, which is owned by a11ybats?

Also I think almost all of the components use this style now -- I think Balloon.V1 might be the only list-of-attributes hold out at this point. Personally I like it better because all the updates are in one place, and it makes it straightforward to have shared patterns across components -- like with `InputErrorAndGuidanceInternal`

Reply:
I hear you on it being confusing though. I'll think about it, for noredink-ui on the whole